### PR TITLE
New Package: ncspot-0.11.2

### PR DESCRIPTION
**Pull request by d33vliter**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/ncspot/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=1
+archs="x86_64"
+wrksrc=${pkgname}-${version}
+create_wrksrc=yes
```
Comment (classabbyamp) [inline srcpkgs/ncspot/template:7]:
```suggestion
```
wrksrc shouldn't be used anymore, and you probably don't need create_wrksrc

File: srcpkgs/ncspot/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=1
+archs="x86_64"
```
Comment (classabbyamp) [inline srcpkgs/ncspot/template:5]: why only x86_64?

File: srcpkgs/ncspot/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+wrksrc=${pkgname}-${version}
+create_wrksrc=yes
+#hostmakedepends=""
+#makedepends=""
+depends="libpulseaudio openssl libxcb dbus"
+short_desc="Is a ncurses Spotify client written in Rust using librespot"
```
Comment (classabbyamp) [inline srcpkgs/ncspot/template:11]:
```suggestion
short_desc="ncurses Spotify client"
```

File: srcpkgs/ncspot/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=1
+archs="x86_64"
+wrksrc=${pkgname}-${version}
+create_wrksrc=yes
+#hostmakedepends=""
+#makedepends=""
+depends="libpulseaudio openssl libxcb dbus"
```
Comment (classabbyamp) [inline srcpkgs/ncspot/template:10]: this should probably be auto-detected if you set makedepends with the proper devel packages

File: srcpkgs/ncspot/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="ncurses Spotify client"
+maintainer="d33vliter <d33vliter@protonmail.com>"
+license="BSD 2-Clause License"
+homepage="https://github.com/hrkfdn/ncspot"
+distfiles="https://github.com/hrkfdn/ncspot/archive/refs/tags/v${version}.tar.gz"
+checksum=0fd9696e13912b906605e9972bf3b2a8a68688cce17478353713550801c94fab
+
+do_install() {
+	cargo build --release
+}
```
Comment (classabbyamp) [inline srcpkgs/ncspot/template:17]:
```suggestion
```
the default do_install from the cargo build_style should work fine here. does it not?

File: srcpkgs/ncspot/template [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=2
+build_style=cargo
+hostmakedepends="pkg-config python3 libpulseaudio"
```
Comment (classabbyamp) [inline srcpkgs/ncspot/template:6]: libpulseaudio should be libpulseaudio-devel and in makedepends

File: srcpkgs/ncspot/template (hunk 5 above)
Comment (classabbyamp) [reply srcpkgs/ncspot/template:17]: https://github.com/void-linux/void-packages/actions/runs/3792212763/jobs/6448498462#step:8:9

do_build is already building the binaries, and do_install is when those files are moved to the package destination directory. with do_install overridden, it creates an empty package

**Comment by icp1994** [comment]
There is also this one: https://github.com/void-linux/void-packages/pull/33600

**Pull request closed without merging**